recipes/mobilesam/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "1.0.0.20231030.12d80d4e32b277de299130d8ce28cc949fb54b6c" %}
+
+package:
+  name: mobilesam
```

Comment:
just falgging this, but the import name of the package is `mobile_sam` the author will likely make a choice on what they name their pypi package, we should try to follow suite.